### ISSUE TITLE
OhGodAnETHlargementPill = LEGIT

### ISSUE THREAD

**Issue by nemosminer** [nemosminer/NemosMiner#547]
Titan (gddr5x) 1080ti 1080 timing mod

testing now

https://github.com/OhGodACompany/OhGodAnETHlargementPill

**Comment by nemosminer** [comment]
https://www.youtube.com/watch?v=-Urx7is1Bls

**Comment by nemosminer** [comment]
confirmed running this with nemosminer claymore on nicehash makes my 1080t's run @50mhs 

Nvidia Inspector OC =  powerlimit=@80tdp(200watt) core+50 mem+450

can get more with fine tuning

**Comment by nemosminer** [comment]
https://www.youtube.com/watch?v=NnOdrA1bCM4

**Comment by borzaka** [comment]
What was the improvements for other algos? Which algo, how much? Thx.

**Comment by nemosminer** [comment]
algos that are memory dependent...

**Comment by nemosminer** [comment]
https://www.youtube.com/watch?v=v9u0wCojZpA

**Comment by nemosminer** [comment]
https://www.youtube.com/watch?v=NAb8LF4QA0U

**Comment by nemosminer** [comment]
https://youtu.be/JMei-i8lPG4

**Comment by borzaka** [comment]
OhGodAnETHlargementPill MINING UPDATE
https://www.youtube.com/watch?v=ic2vdYe1sk8

**Comment by nemosminer** [comment]
https://www.youtube.com/watch?v=ZLTRYp_kCYg

interview with developer

**Comment by nemosminer** [comment]
they also have private version for ALL GTX CARDS

**Comment by borzaka** [comment]
So, is it worth to mine ethash with a 1080Ti RIG with this pill, or there are more profitable algos?

**Comment by nemosminer** [comment]
how long is a peice if string? 

why dont' you test for yourself

**Comment by nemosminer** [comment]
i run the pill at startup so it always running in background on my 6x1080ti rigs it helps other algo not just claymore and eth...

**Comment by borzaka** [comment]
I thought so. Yes, I tested for myself as well, but I have a mixed RIG (3x 1080 Ti, 3x 1070 Ti), and I had to separate them to run benchmark only on the 1080 Tis with and without the PILL...

I was able to get 55,5 MH/s on all my 1080 Ti, but the earnings was not so good as other algos.

Yes, it helps some other algos, but some others produced worse hashrate with the PILL. For example x16r/s is very fluctuating.
